server/store.go

```diff
@@ -123,6 +135,7 @@ func (s *storeInfo) getLocationID(keys []string) string {
 type StoreStatus struct {
 	*pdpb.StoreStats
 
+	Blocked           bool      `json:"blocked"`
```

Comment:
or maybe `holdingScheduler string`, which means a scheduler is running exclusively, and we can use `holdScheduler`, `unholdScheduler`, `isSchedulerHolded` too. 

/cc @disksing @huachaohuang